lib/index.js

```diff
@@ -52,6 +52,9 @@ function exec(command, ...args) {
   return deferred.promise;
 }
 
+const endsWith = (suffix, str) =>
+  str.indexOf(suffix, str.length - suffix.length) !== -1;
+
```

Comment:
Sure it is caused by that. If we don't pollute the global namespace there is no way each string could be extended at runtime. (Technically it could but that would involve some nasty wrapping of each string)